samples/ResourceOwnerPasswordFlow/DeveloperNotes/ConformanceProfiles/fiddler-request-OP-Response-Missing.txt

```diff
@@ -0,0 +1,7 @@
+﻿POST http://localhost:50000/connect/token HTTP/1.1
```

Comment:
I think you're right, the JS app should be enough (even for those who don't use Fiddler).